java/runtime/src/main/resources/ray.default.conf

```diff
@@ -56,53 +53,28 @@ ray {
     max-backup-files: 10
   }
 
-  // Custom worker jvm parameters.
-  worker.jvm-parameters: []
-
-  // Custom `java.library.path`
-  // Note, do not use `dir1:dir2` format, put each dir as a list item.
-  library.path: []
-
-  // Custom classpath.
-  // Note, do not use `dir1:dir2` format, put each dir as a list item.
-  classpath = []
-
   // ----------------------
   // Redis configurations
   // ----------------------
   redis {
-    // If `redis.server` isn't provided, which port we should use to start redis server.
-    // If `head-port` is not provided, it will be generated randomly.
-    // head-port: 6379
-    // Below passwords should be consistent with the one defined in python/ray/ray_constants.py.
-    // The password used to start the redis server on the head node.
-    head-password: "5241590000000000"
     // The password used to connect to the redis server.
     password: "5241590000000000"
-    // If `redis.server` isn't provided, how many Redis shards we should start in addition to the
-    // primary Redis shard. The ports of these shards will be `head-port + 1`, `head-port + 2`, etc.
-    shard-number: 1
-  }
-
-  // ----------------------------
-  // Object store configurations
-  // ----------------------------
-  object-store {
-    // Initial size of the object store.
-    size: 10 MB
   }
 
   // ----------------------------
   // Raylet configurations
   // ----------------------------
   raylet {
     // See src/ray/ray_config_def.h for options.
+    // Below section takes effect only if Ray head is started by a driver.
     config {
       // TODO(zhuohan): enable this for java
       put_small_object_in_memory_store: false
     }
   }
 
-  // Whether we enable job manager to submit and manage job.
-  enable-job-manager: false
+  // Below section takes effect only if Ray head is started by a driver.
+  head {
+    args: []
```

Comment:
1) Document that this will be used for `ray start`. 
2) `head-args` may look better.